edgelet/edgelet-http-mgmt/Cargo.toml

```
@@ -16,7 +16,7 @@ percent-encoding = "2"
 regex = "1"
 serde = "1"
 serde_json = "1"
-tokio = { version = "1", features = ["macros", "sync"] }
+tokio = { version = "1", features = ["macros", "parking_lot", "sync"] }
```

```suggestion
tokio = { version = "1", features = ["parking_lot", "rt", "sync"] }
```
This crate uses `tokio::spawn`, which requires `rt`. `macros` can be in the development dependencies.